apps/chat/src/components/Settings/CustomLogoSelect.tsx

```diff
@@ -59,7 +59,7 @@ export const CustomLogoSelect = ({
         <FileManagerModal
           isOpen
           allowedTypes={['image/*']}
-          allowedTypesLabel="images"
+          allowedTypesLabel={t('images')}
```

Comment:
can be removed in case if we show the same in `FileManagerModal.tsx`

Reply:
Thank you!
Removed.